plugins/woocommerce-admin/client/products/sections/images-section.tsx

```diff
@@ -185,89 +179,50 @@ export const ImagesSection: React.FC = () => {
-											filesList:
-												currentTarget.files as FileList,
-											onError: () => null,
-											onFileChange: onFileUpload,
-											maxUploadFileSize: 10000000,
-										} );
+								<MediaUploader
+									onError={ () => null }
+									onFileUploadChange={ onFileUpload }
+									onSelect={ ( file ) => {
+										if (
+											images.find(
+												( img ) => file.id === img.id
+											) === undefined
+										) {
+											recordEvent(
+												'product_images_add_via_media_library'
+											);
+											setValue( 'images', [
+												...images,
+												file,
+											] );
+										}
+									} }
+									onUpload={ ( files ) => {
+										if ( files[ 0 ].id ) {
+											recordEvent(
+												'product_images_add_via_drag_and_drop_upload'
+											);
+											setValue( 'images', [
+												...images,
+												...files,
+											] );
+										}
 									} }
-									render={ ( { openFileDialog } ) => (
-										<div
-											className="woocommerce-form-file-upload"
-											onKeyPress={ () => {} }
-											tabIndex={ 0 }
-											role="button"
-											onClick={ (
-												event: React.MouseEvent<
-													HTMLDivElement,
-													MouseEvent
-												>
-											) => {
-												const { target } = event;
-												if (
-													(
-														target as HTMLButtonElement
-													 )?.type !== 'button'
-												) {
-													openFileDialog();
-												}
-											} }
-											onBlur={ () => {} }
-										>
-											<MediaUploader
-												onError={ () => null }
-												onSelect={ ( file ) => {
-													if (
-														images.find(
-															( img ) =>
-																file.id ===
-																img.id
-														) === undefined
-													) {
-														recordEvent(
-															'product_images_add_via_media_library'
-														);
-														setValue( 'images', [
-															...images,
-															file,
-														] );
-													}
-												} }
-												onUpload={ ( files ) => {
-													if ( files[ 0 ].id ) {
-														recordEvent(
-															'product_images_add_via_drag_and_drop_upload'
-														);
-														setValue( 'images', [
-															...images,
-															...files,
-														] );
-													}
-												} }
-												label={
-													<>
-														<img
-															src={ DragAndDrop }
-															alt="Completed"
-															className="woocommerce-product-form__drag-and-drop-image"
-														/>
-														<span>
-															{ __(
-																'Drag images here or click to upload',
-																'woocommerce'
-															) }
-														</span>
-													</>
-												}
+									label={
+										<>
+											<img
+												src={ DragAndDrop }
+												alt="Completed"
```

Comment:
I know this was from the original changes, but should this be wrapped in `__(`?

Reply:
Nice catch @louwie17!
I just pushed this fix. Could you take another quick look at this PR?